centreon/src/Core/Host/Application/UseCase/AddHost/AddHost.php

```diff
@@ -118,7 +120,9 @@ public function __invoke(AddHostRequest $request, AddHostPresenterInterface $pre
                 $this->linkHostGroups($request, $hostId);
                 $this->linkParentTemplates($request, $hostId);
                 $this->addMacros($request, $hostId);
-                // Note: host is not linked to any ACLsResource
```

Comment:
Are you sure to delete this comment ?

Reply:
Yep, the code that follows link the host to an ACLsResource